src/D2L.CodeStyle.Analyzers/ApiUsage/System.Collections.Immutable/ImmutableCollectionsAnalyzer.cs

```diff
@@ -14,9 +14,10 @@ public sealed class ImmutableCollectionsAnalyzer : DiagnosticAnalyzer {
 
 		public override void Initialize( AnalysisContext context ) {
 			context.EnableConcurrentExecution();
-			context.RegisterCompilationStartAction( RegisterAnalysis );
-
+			// TODO: fix analyzer
+			//context.RegisterCompilationStartAction( RegisterAnalysis );

```

Comment:
Looks like you changed the analyzer behaviour but not the test expectations, which caused CI to fail.

Is there a way to have an analyzer exist in the DLL, but not run it against the LMS?  Otherwise, we might have to turn off the tests as well.

Reply:
I switched to a straight revert. This analyzer is interesting but not critical at the moment.